napalm/junos/junos.py

```diff
@@ -403,11 +403,18 @@ def get_interfaces_counters(self):
         """Return interfaces counters."""
         query = junos_views.junos_iface_counter_table(self.device)
         query.get()
+        query_logical = junos_views.junos_logical_iface_counter_table(self.device)
```

Comment:
I don't think we should make a separate request just for this: the logical interfaces are already present in the `junos_iface_counter_table`, we just need to extract them. For this we can use the nested tables junos-eznc offers, see an example [here](https://github.com/napalm-automation/napalm/blob/develop/napalm/junos/utils/junos_views.yml#L64-L75) where a view is able to reference a  sub-table. In the same way, we can have the logical interfaces nested under a sub-table, and avoid an additional RPC request.